baybe/surrogates/custom.py

```diff
@@ -149,7 +79,7 @@ def default_model(self) -> ort.InferenceSession:
             raise ValueError("Invalid ONNX string") from exc
 
     @batchify
-    def _posterior(self, candidates: Tensor) -> tuple[Tensor, Tensor]:
+    def _estimate_moments(self, candidates: Tensor) -> Posterior:
```

Comment:
base class says return type is `tuple[Tensor, Tensor]:`

Reply:
thanks, overlooked, but now fixed!